scripts/model/repo.js

```diff
@@ -0,0 +1,22 @@
+(function(module) {
+  var repos = {};
+
+  repos.all = [];
+
+  repos.requestRepos = function(callback) {
+    $.getJSON('https://api.github.com/users/vbenavente/repos' +
+        '?per>page5' +
+        '&sort=updated', function(response) {
+      repos.all = response;
+    });
+    callback();
```

Comment:
You should consider moving your callback up into the function that runs on success.  The way you currently have it can cause the callback to be called before you get the response back.